### ISSUE TITLE
Update WMCore central services specs to python3

### ISSUE THREAD

**Issue by amaltaro** [dmwm/WMCore#10418]
**Impact of the new feature**
All the WMCore services but WMAgent

**Is your feature request related to a problem? Please describe.**
Once we have sorted all the WMCore dependencies and are confident with migrating central services to python3, we will need to update their spec files and **stop** building their python2 version, build and uploads will then start to be made only against the python3 stack

**Describe the solution you'd like**
Update all the WMCore central services spec files and migrate all their dependencies to python3 stack.
An example list of those services can be seen in this PR: https://github.com/cms-sw/cmsdist/pull/6768

**Describe alternatives you've considered**
An alternative would be to create new spec files for the python3 stack, but then we would have to adapt our deployment scripts as well, thus injecting a reasonable amount of extra work.

**Additional context**
none

**Comment by amaltaro** [comment]
I think we can call this issue done (even though it should remain open until we actually merge things in cmsdist).

I have created a new cmsdist PR:
https://github.com/cms-sw/cmsdist/pull/6982

where the WMCore central services specs have all been converted to python3. Services are not yet functional because we know there are still quite a few issues with our source code, but I managed to test the:
* build process
* upload to cmsrep
* services deployment in a VM (including the `admin` and `frontend` packages)
* and the execution of all those services (all returning a green RUNNING status)

We will need this deployment change though (pystack is no longer relevant in python3): https://github.com/dmwm/deployment/pull/1056

and WMCore services 1.4.9.pre6 need to be patched with: https://github.com/dmwm/WMCore/pull/10573
otherwise even a `manage status` command would fail with ImportError from a wrong python interpreter.

I'm moving this issue to the Waiting column, until we can update those spec versions and get a first version deployed in testbed.
